### PR TITLE
Update brilliant-mirabella-genio-smart-plugs.rst - Added Gosund…

### PR DESCRIPTION
**Pull request by Taigar2015**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Comment by Taigar2015** [comment]
@OttoWinter I understand I need to change the lengt of the bar under the title and the esphome.io URL. But how do I do this?
This is my second pull request, I still have no idea..

File: cookbook/brilliant-mirabella-genio-smart-plugs.rst [hunk 1]
```diff
@@ -216,6 +216,74 @@ which these adaptions created by `@cryptelli <https://community.home-assistant.i
         pin: GPIO12
         id: relay
 
+
+3.3 Gosund SPI
+***************************************
```
Comment (Taigar2015) [inline cookbook/brilliant-mirabella-genio-smart-plugs.rst]: This line needs to be the same length as the title

File: cookbook/brilliant-mirabella-genio-smart-plugs.rst [hunk 2]
```diff
@@ -216,6 +216,74 @@ which these adaptions created by `@cryptelli <https://community.home-assistant.i
...
+          name: "Gosund SP1 Smart Plug Voltage"
+          unit_of_measurement: V
+        power:
+          name: "Gosund SP1 Smart Plug Wattage"
+          unit_of_measurement: W
+          id: "energy_temp_Wattage"
+        change_mode_every: 8
+        update_interval: 10s
+
+Check the following page for calibrating the measurements: https://esphome.io/components/sensor/index.html#calibrate-linear
```
Comment (Taigar2015) [inline cookbook/brilliant-mirabella-genio-smart-plugs.rst]: This url needs to be relative, so without the esphome.io part.

**Comment by OttoWinter** [comment]
@Taigar2015 Go to https://github.com/Taigar2015/esphome-docs/tree/patch-4, browser through the files there and use the edit button to make changes (commit directly to patch-4 branch when asked).

**Comment by Taigar2015** [comment]
@OttoWinter Thanks, now I know where to change. I also found out how to get to that page. So next time I will find it myself.

I changed the length under the title, but the hyperlink isn't working yet. Tried different things, but didn't find the right one.
Did a lot of Googling on this, but it didn't help.

Can you tell me how I should enter the URL?

**Comment by OttoWinter** [comment]
```rst
:ref:`sensor-filter-calibrate_linear`
```

Also see contributing guide.

**Review by OttoWinter — APPROVED** [review]

File: cookbook/brilliant-mirabella-genio-smart-plugs.rst [hunk 3]
```diff
@@ -216,6 +216,74 @@ which these adaptions created by `@cryptelli <https://community.home-assistant.i
...
+      - platform: hlw8012
+        sel_pin:
+          number: GPIO12
+          inverted: True
+        cf_pin: GPIO04
+        cf1_pin: GPIO05
+        current_resistor: 0.00221
+        voltage_divider: 871
+        current:
+          name: "Gosund SP1 Smart Plug current"
```
Comment (brandond) [inline cookbook/brilliant-mirabella-genio-smart-plugs.rst:273]: maybe capitalize Current like you did Voltage and Wattage?

**Pull request merged**